renovate.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": [
+    "config:base"
+  ]
+}
```

Comment:
What is `renovate.json` for?

Reply:
This should be automatically generated by Renovate bot when it's enabled